### ISSUE TITLE
Can values truly be null?

### ISSUE THREAD

**Issue by ljnelson** [microbean/microbean-helm#17]
Here is a representative stack trace:
```java
Caused by: java.lang.NullPointerException
	at java.util.Objects.requireNonNull(Objects.java:203)
	at org.microbean.helm.chart.Values.coalesceMaps(Values.java:155)
	at org.microbean.helm.chart.Configs.computeEffectiveValues(Configs.java:205)
	at org.microbean.helm.chart.Configs.computeEffectiveValues(Configs.java:189)
	at org.microbean.helm.chart.Configs.toValuesMap(Configs.java:236)
	at org.microbean.helm.chart.Configs.toValuesMap(Configs.java:137)
	at org.microbean.helm.chart.Configs.toDefaultValuesMap(Configs.java:93)
	at org.microbean.helm.chart.Requirements.processSingleChartImportValues(Requirements.java:207)
	at org.microbean.helm.chart.Requirements.processImportValues(Requirements.java:195)
	at org.microbean.helm.chart.Requirements.apply(Requirements.java:476)
	at org.microbean.helm.chart.Requirements.apply(Requirements.java:364)
	at org.microbean.helm.ReleaseManager.install(ReleaseManager.java:299)
```

**Comment by ljnelson** [comment]
The problem is that [`Yaml#load(String)` can return `null`](https://static.javadoc.io/org.yaml/snakeyaml/1.19/org/yaml/snakeyaml/Yaml.html#load-java.lang.String-).  See [line 204 in `Configs.java`](https://github.com/microbean/microbean-helm/blob/v2.6.2.1.1.0/src/main/java/org/microbean/helm/chart/Configs.java#L203-L204).

**Comment by ljnelson** [comment]
Fixed by #18.
